sdk/storage/storage-file/src/policies/SharedKeyCredentialPolicy.ts

```diff
@@ -76,6 +76,10 @@ export class SharedKeyCredentialPolicy extends CredentialPolicy {
       `SharedKey ${this.factory.accountName}:${signature}`
     );
 
+    // Workaround for https://github.com/axios/axios/issues/2107
+    // We should keep the 'content-length' header once the issue is solved
+    request.headers.remove(HeaderConstants.CONTENT_LENGTH);
```

Comment:
@kinelski 
So, the record and playback PR https://github.com/Azure/azure-sdk-for-js/pull/2477 for storage-file assumes this change already ?

Reply:
Yes. I have tested it with this change and it works. Previously I was omitting the Axios code for testing.

Reply:
Update: I have realized only playback was tested. I'll make sure recording and default work as well.

Reply:
Just added a "small" fix. All tests work (default, record, playback).